src/main/java/org/folio/event/service/UserOutboxService.java

```diff
@@ -54,67 +54,76 @@ public UserOutboxService() {
    * @return future with integer how many records have been processed
    */
   public Future<Integer> processOutboxEventLogs(Vertx vertx, Map<String, String> okapiHeaders) {
+    logger.debug("processOutboxEventLogs:: Trying to process outbox event logs");
     String tenantId = TenantTool.tenantId(okapiHeaders);
     PostgresClient pgClient = PostgresClient.getInstance(vertx, tenantId);
     return pgClient.withTrans(conn -> lockRepository.selectWithLocking(conn, OUTBOX_LOCK_NAME, tenantId)
       .compose(retrievedCount -> outboxRepository.fetchEventLogs(conn, tenantId))
       .compose(logs -> {
         if (CollectionUtils.isEmpty(logs)) {
+          logger.warn("processOutboxEventLogs:: OutBoxEventLogs is empty");
```

Comment:
I don't this it is warn level, because these endpoint to process outbox logs executed also by timer once at 30 mins, and it is expected that it can not find outbox logs, so better to change to info

Reply:
changed to info